app/components/view-epics/macro.test.js

```diff
@@ -0,0 +1,458 @@
+import request from 'supertest';
+import cheerio from 'cheerio';
+import { createTestHarness } from '../../testUtils/testHarness';
+
+const macroWrapper = `{% from './components/view-epics/macro.njk' import viewEpics %}
+                        {{ viewEpics(params) }}`;
+
+describe('view-epics', () => {
+  describe('for must epics', () => {
+    it('should render the tag if data provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {
+            must: {
+              met: [
+                {
+                  id: 'C14E1',
+                  name: 'access prescribable items',
+                },
+              ],
+              'not-met': [
+                {
+                  id: 'C14E13',
+                  name: 'access Patient Record',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="must-tag"]').text().trim()).toEqual('Must Epics');
+
+          done();
+        });
+    });
+
+    it('should render the data if provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {
+            must: {
+              met: [
+                {
+                  id: 'C14E1',
+                  name: 'access prescribable items',
+                },
+                {
+                  id: 'C14E2',
+                  name: 'manage Formularies',
+                },
+              ],
+              'not-met': [
+                {
+                  id: 'C14E13',
+                  name: 'access Patient Record',
+                },
+                {
+                  id: 'C14E14',
+                  name: 'search the directory',
+                },
+              ],
+            },
+          },
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="must-epics"]').length).toEqual(1);
+
+          done();
+        });
+    });
+
+    it('should not render the data if not provided', (done) => {
+      const context = {
+        params: {
+          dataTestId: 'some-data-identifier',
+          data: {},
+        },
+      };
+
+      const dummyApp = createTestHarness(macroWrapper, context);
+      request(dummyApp)
+        .get('/')
+        .then((res) => {
+          const $ = cheerio.load(res.text);
+          expect($('[data-test-id="must-epics"]').length).toEqual(0);
+
+          done();
+        });
+    });
+
+    describe('with met type', () => {
+      it('should render the data if provided', (done) => {
```

Comment:
I would change this to `should render the viewEpicList component when provided with epics that are must and met`